OracleDatabase/SingleInstance/dockerfiles/buildDockerImage.sh

```diff
@@ -112,6 +112,9 @@ elif [ $EXPRESS -eq 1 ]; then
   elif [ "$VERSION" == "11.2.0.2" ]; then
     EDITION="xe"
     DOCKEROPS="--shm-size=1G $DOCKEROPS";
+  elif [ "$VERSION" == "18.4.0" ]; then
```

Comment:
This file looks out of date with what's in the master.

Reply:
Your PR is out of date. The files have changed in the meantime and you need to do a fresh merge from `master` before we can merge this.
For example, your `buildDockerImage.sh` file does not reflect the change yet that the Dockerfiles have been merged for 12.2.0.1 and 18.3.0.

Meaning in the one file from the master you will find this at line [138 onwards](https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/dockerfiles/buildDockerImage.sh#L138-L141):
```
# Which Dockerfile should be used?
if [ "$VERSION" == "12.1.0.2" ] || [ "$VERSION" == "11.2.0.2" ]; then
  DOCKERFILE="$DOCKERFILE.$EDITION"
fi;
```

Your file doesn't have that change present yet.
Please merge from `oracle/master` again to make sure that we are not losing any changes from a stale PR.